### ISSUE TITLE
Accessibility: add label under icons on mobile navbar

### ISSUE THREAD

**Issue by Dehelssey** [elk-zone/elk#3119]
Accessibility best practices states that mobile navbar items should have a label.

We should at least offer an option to display them. 

Or, even better, display them by default with an option to hide them

**Comment by userquin** [comment]
We have the aria-label, we should add aria-hidden="true" to the icons div/span: do you mean adding a new option to display de text, something like this?

![Image](https://github.com/user-attachments/assets/442fb002-bdb6-4ea5-9ffa-a80dbd8e7f6b)
_https://m3.material.io/components/navigation-bar/overview_

**Comment by Dehelssey** [comment]
Yeah, this exactly. 

I assumed the labels were in the DOM but hidden in CSS on mobile so yeah we should have them visible (idealy by default with an option to hide them as they are now)

**Comment by shuuji3** [comment]
I tried implementing this label by #3119. Any feedback is welcome 🙂
